### ISSUE TITLE
internet connection timeout

### ISSUE THREAD

**Issue by lupa18** [EvanRespaut/Equate#35]
Since some days I detected something not working with currency conversion.

And or gives me "internet connection timeout" error
![img_20180913_162750](https://user-images.githubusercontent.com/463429/45510940-11f19580-b772-11e8-9dec-7d2c7c1b080a.jpg)

I tried different networks

I'm using 1.6 version

**Comment by sebadamus** [comment]
Justo tried 1.7 from playstore (used to get you from fdroid), seems new api dont have the same currencies avaiable :-(, south america i.e., hope it can be added later, thanks.

**Comment by lupa18** [comment]
yes, I'm in south america 
this was a very powerful tool for me :(

**Comment by EvanRespaut** [comment]
Yeah the new source has less sources, sorry about that.  I can look around for another API that provides more currencies, but options that are free are very limited :(

**Comment by EvanRespaut** [comment]
Just updated the app to add back support for ~150 currencies! Let me know if people have issues
